Formula/theharvester.rb

```diff
@@ -26,6 +26,10 @@ class Theharvester < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
 
+  # How to update the resources
+  # tar -zxvf theHarvester-4.0.3.tar.gz
+  # cd theHarvester-4.0.3 && pip install -r requirements/base.txt
+  # run homebrew-pypi-poet on the freezed dependencies
```

Comment:
How do they suddenly become frozen dependencies here? Generally pip dependencies are not frozen.

Reply:
yeah, I had to run the pip freeze in the virtualenv setup.